### ISSUE TITLE
Show selected notes on fretboard

### ISSUE THREAD

**Issue by tomasz13** [dmitrio95/fretboard-plugin#2]
it would be nice to see every selected notes on the fretboard visualisation...

something like this:
![selectednotes](https://user-images.githubusercontent.com/12948865/76405312-3ab71300-6388-11ea-9364-04f35ff833db.jpg)

thank you

**Comment by dmitrio95** [comment]
Indeed, the case of a range selection was implemented recently in 35c84dc8632cb29a04e65bcfa208020a49fe14fa, so the plugin now displays all notes in a range selection. Maybe this should be extended for a non-range selection containing multiple notes.

I wonder what would be the expected behavior though: if one note is selected the fretboard displays its entire chord (which is expected for using the fretboard widget as a note input tool). Does this mean that if two notes are selected then both chords should be displayed on the fretboard? Or just these two notes? And what is the scenario when such handling of multiple notes selection is needed?
